ast/src/analyzed/mod.rs

```diff
@@ -927,6 +927,54 @@ impl<T> Children<AlgebraicExpression<T>> for LookupIdentity<T> {
     }
 }
 
+/// A witness generation helper for a lookup identity. This is ignored by the backend.
```

Comment:
Maybe add: This identity is commonly used as a replacement for a lookup identity that has been converted to challenge-based polynomial identities.

Reply:
done